Resources/Prototypes/_CP14/Entities/Objects/Tools/flashlight.yml

```diff
@@ -0,0 +1,40 @@
+- type: entity
+  name: Лазурный фонарь
+  parent: BaseItem
+  id: CPFlashlightLantern
+  description: Пережиток прошлого техномагии. Большой, тяжелый, непрактичный. Таким приятно разве что бить по голове.
+  components:
+  - type: Sprite
+    sprite: _CP14/Objects/Tools/magic-lamp.rsi
+    noRot: false
+    layers:
+    - state: lamp    
+  - type: MeleeWeapon
+    attackRate: 0.7
+    wideAnimationRotation: 10
+    wideAnimation: CP14WeaponArcSlash
+    damage:
+      types:
+        Blunt: 8
+    soundHit:
+      collection: MetalThud
+    cPAnimationLength: 0.25
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/Items/toolbox_drop.ogg
+  - type: Item
+    sprite: _CP14/Objects/Tools/magic-lamp.rsi
+    size: Large
+    storedRotation: -90
+  - type: Rotatable
+  - type: PointLight
+    mask: /Textures/Effects/LightMasks/double_cone.png
+    color: "#47F8FFFF"
+    autoRot: true
+    energy: 5
+    radius: 8
+    netsync: false
+  - type: Appearance
```

Comment:
Also i think it should be toggleable. Need ability to turn on/turn off the light

Reply:
Nah. The Idea is that this is old shitty flashlight. It so old and bad that it can not even be toggled off besides it is large, must be carried in hand. By the way you can crush skulls with it

Reply:
it's too bright and useful as a lighting source to go under the “old and useless” category

Reply:
The point is that this is indeed useful as light source, but you always must carry this in in hand, it takes a lot of space in storage, and if you wanna hide you can't just switch off it, you must put that in back pack which takes a quite amount of time. I'd like even maybe give it much bigger size then it has now and reduce size a lil bit

Reply:
uh, radius i mean